src/main/resources/standardTestCases/monitoring_log.test

```diff
@@ -0,0 +1,18 @@
+# Test to confirm the monitoring log file is created and the monitoring script contents are transferred into it.
```

Comment:
The monitoring script is supposed to produce the contents of the monitoring log file, no?